### PR TITLE
Add several spelling corrections + refinements

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -6948,6 +6952,7 @@ compilaton->compilation
 compilatons->compilations
 compilcate->complicate
 compilcated->complicated
+compilcates->complicates
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:6955]:
```text
compilcatedly->complicatedly
compilcating->complicating
compilcation->complication
compilcations->complications
```
* https://github.com/search?q=compilcatedly&type=code
* https://github.com/search?q=compilcating&type=code
* https://github.com/search?q=compilcation&type=code
* https://github.com/search?q=compilcations&type=code

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -7660,6 +7665,8 @@ consants->constants
 conscent->consent
 consciencious->conscientious
 consciouness->consciousness
+consctructor->constructor
+consctructors->constructors
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:7669]:
```
consctruct->construct
consctructed->constructed
consctructing->constructing
consctruction->construction
consctructions->constructions
consctructive->constructive
consctructs->constructs
```
* https://github.com/search?q=consctruct&type=code
* https://github.com/search?q=consctructed&type=code
* https://github.com/search?q=consctructing&type=code
* https://github.com/search?q=consctruction&type=code
* https://github.com/search?q=consctructions&type=code
* https://github.com/search?q=consctructive&type=code
* https://github.com/search?q=consctructs&type=code

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -7923,7 +7930,7 @@ containa->contain
 containe->contain, contained, container, contains,
 containees->containers
 containerr->container
-containes->contains
+containes->contains, container,
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:7933]: Also suggest `contained` as `d` key is beside `s`
```text
containes->contains, container, contained,
```

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -11685,6 +11694,8 @@ distingishes->distinguishes
 distingishing->distinguishing
 distingiush->distinguish
 distingquished->distinguished
+distinguise->distinguish
+distinguised->distinguished
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:11698]:
```text
distinguises->distinguishes
```
* https://github.com/search?q=distinguises&type=code

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -15019,6 +15034,7 @@ featurs->features
 feautre->feature
 feauture->feature
 feautures->features
+febbruary->february
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:15037]: If you don't mind capitalised with `February`. Online dictionaries, browser, aspell, hunspell, ispell indicates a misspelling if it is in lower case. Thank you, if you don't mind adding these additional related 'months'.
`Februar` should be German, remove it if you think we should not add it.
```text
febbruary->February
februar->February

augest->August
novemeber->November
```
* https://github.com/search?q=febbruary&type=code
* https://github.com/search?q=februar&type=code
* https://github.com/search?q=augest&type=code
* https://github.com/search?q=novemeber&type=code

**Review by vikivivi — CHANGES_REQUESTED** [review]

**Comment by luzpaz** [comment]
Thanks, I was sloppy on this PR. Apologies. I've added your revisions and also made some of my own: [eb3c962](https://github.com/codespell-project/codespell/pull/2441/commits/eb3c962edf773c05a1ae716216365ba067e493c8)

**Review by vikivivi — APPROVED** [review]
LGTM

**Pull request merged**